integration-tests/field-validation/src/index.exo

```diff
@@ -5,5 +5,7 @@ module ExampleDatabase {
     @pk id: Int = autoIncrement()
     @range(min = 0, max = 2) limit: Int?
     age: Int?
+    @range(min = "-1.0", max = "3.0") flimit: Float?
+    price: Float?
```

Comment:
Maybe add another Float field to ensure that our impl differentiates same-typed fields?

While at it, I generally try to name types such as `Something` to be more meaningful: `Concert`, `Todo`, etc. In the long run, they convey the intention and the purpose of the feature more directly.